modules/entity/src/sorted_state_adapter.ts

```diff
@@ -106,24 +106,21 @@ export function createSortedStateAdapter<T>(selectId: any, sort: any): any {
     }
   }
 
-  function upsertOneMutably(update: Update<T>, state: R): DidMutate;
-  function upsertOneMutably(update: any, state: any): DidMutate {
-    return upsertManyMutably([update], state);
+  function upsertOneMutably(entity: T, state: R): DidMutate;
+  function upsertOneMutably(entity: any, state: any): DidMutate {
+    return upsertManyMutably([entity], state);
   }
 
-  function upsertManyMutably(updates: Update<T>[], state: R): DidMutate;
-  function upsertManyMutably(updates: any[], state: any): DidMutate {
+  function upsertManyMutably(entities: T[], state: R): DidMutate;
+  function upsertManyMutably(entities: any[], state: any): DidMutate {
     const added: T[] = [];
     const updated: Update<T>[] = [];
 
-    for (const update of updates) {
-      if (update.id in state.entities) {
-        updated.push(update);
+    for (const entity of entities) {
+      if (entity.id in state.entities) {
```

Comment:
You can't safely access `entity.id`. You will need to use the adapter's `selectId` function to get it.

Reply:
Right, totally forgot about it, sorry :sweat_smile: 
I think some tests on the state adapters with a different `selectId` implementation would prevent these mistakes. But I don't know if these are worth the effort? Or maybe rename `id` on the book fixture?